Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Components/EmpArtifactComponent.cs

```diff
@@ -0,0 +1,14 @@
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Components;
+
+/// <summary>
+/// Artifact that EMP
+/// </summary>
+[RegisterComponent]
+public sealed class EmpArtifactComponent : Component
+{
+    [DataField("range")]
```

Comment:
public fields in components need datafields. Yes i know it's obvious; it still needs to be done.

Reply:
Fun fact: I took it from ignite artifact, so you may want to fix it there